tests/ti/cc3220_launchpad/common/application_code/ti_code/network_if.c

```diff
@@ -58,6 +58,12 @@
 #include "network_if.h"
 #include "uart_term.h"
 
+/* Redefine UART_PRINT to no-op so that test output is not clobbered. */
```

Comment:
You're changing the UART_PRINT macro in uart_term.h, which is included by this module, as part of this PR. So why add these lines here?

Also, if anybody falls into this block of code with the device prompting the developer to type in a new SSID, they're going to be confused because there won't be any prompt text.

Reply:
We **changed** the `UART_PRINT` print macro in uart_term.h before, this commit reverts them. If you compare the file with the one in demos code, they look the same now.

As to the second point, this is test code, which is only for vendor who are qualifying a new board and running our tests. That's why we disabled the `UART_PRINT` macro before because it's causing problems with our test automation. This commit only fixed the warning problems we introduced before.